sections/dom.include

```diff
@@ -604,140 +604,75 @@
-    especially common with speech browsers, where users would otherwise find quickly navigating a
-    page quite difficult.
-
-    Even beyond browsers, software can make use of this information. Search engines can use the
-    headings to more effectively index a page, or to provide quick links to subsections of the page
-    from their results. Tools can use the headings to create a table of contents (that is in fact
-    how this very specification's table of contents is generated).
-
-    This example has focused on headings, but the same principle applies to all of the semantics
-    in HTML.
-  </div>
-
-  Authors must not use elements, attributes, or attribute values for purposes other than their
-  appropriate intended semantic purpose, as doing so prevents software from correctly processing the
-  page.
+</div>
+
+This semantic information is critical to assistive technologies. For example, a screen reader will query the browser for semantic information and use that information to present the document or application in synthetic speech.
+
+In some cases assistive technologies use semantic information to provide additional functionality. A speech recognition tool might provide a voice command for moving focus to the start of the main element for example.
+
+When the appropriate HTML element or attribute is not used, it deprives HTML processors of valuable semantic information.
+
+<div class="example">
+<p>In this example the visual manifestation of the document may look the part, but because the appropriate HTML elements have not been used there is little semantic information available to browsers, search engines or assistive technologies.</p>
+
+<pre highlight="html">
+    &lt;!doctype html&gt;
+&lt;html lang="en"&gt;
+  &lt;head&gt;
+    &lt;title&gt;Favorite books&lt;/title&gt;
+  &lt;/head&gt;
+  &lt;body&gt;
+    &lt;div class="header"&gt;
+      &lt;img src="logo.png" alt="Favorite books logo"&gt;
+    &lt;/div&gt;
+    &lt;div class="main"&gt;
+      &lt;span class="largeHeading"&gt;Favorite books&lt;/span&gt;
+      &lt;p&gt;These are a few of my favorite books.&lt;/p&gt;
+      &lt;span class="smallHeading"&gt;The Belgariad&lt;/span&gt;
+      &lt;p&gt;Five books by David and Leigh Eddings.&lt;/p&gt;
+      &lt;span class="smallHeading"&gt;The Hitchhiker's Guide to the Galaxy&lt;/span&gt;
+      &lt;p&gt;A trilogy of five books by Douglas Adams.&lt;/p&gt;
+    &lt;/div&gt;
+  &lt;/body&gt;
+&lt;/html&gt;
+</pre>
+</div>
+
+A document can change dynamically whilst it is being processed. Scripting and other mechanisms can be used to change attribute values, text, or the entire document structure. The semantics of a document are therefore based on the document's state at a particular instance in time, but may also change in response to external events. User agents <em>must</em> update their presentation of the document to reflect these changes.
```

Comment:
I don't think we need to emphasize the word "must". However it might be a good issue to raise to identify all the RFC2119 keywords and emphasize them in some way.